.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,19 @@
-# Pull Request
-
-## [Link to story in trello](#)
+# [Story in Trello](#)
 
 ## Done?
 
-- [ ] Acceptance criterias are met
-- [ ] Code quality checks are green
-- [ ] Readme updated if needed
-- [ ] Story under test
-- [ ] Mobile works
-- [ ] Seeds created
-- [ ] Translated
-- [ ] Code is reviewed
+### Done for approval?
+
+* [ ] Acceptance criteria are met
+* [ ] Code quality checks are green
+* [ ] Readme updated if needed
+* [ ] Story under test
+* [ ] Mobile works
+* [ ] Seeds created
+* [ ] Translated
+* [ ] Code is reviewed
```

Comment:
Its basicly the same checkboxes as the actual template allready has, just using the right list markers for GitHubs Markdown UL's. The Checkboxes are rather considered as a reminder and checklist, not a mendatory thing to have them checked.